### PR TITLE
Log and skip adapter ping error when retrieve adapter capability

### PR DESCRIPTION
**Pull request by stonezdj**
Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [ ] Well Written Title and Summary of the PR
- [ ] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [ ] Accepted the DCO. Commits without the DCO will delay acceptance.
- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by zyyw — APPROVED** [review]

**Review by wy65701436 — APPROVED** [review]
lgtm

File: src/controller/scanner/base_controller.go [hunk 1]
```diff
@@ -80,8 +80,9 @@ func (bc *basicController) ListRegistrations(ctx context.Context, query *q.Query
 		return nil, errors.Wrap(err, "api controller: list registrations")
 	}
 	for _, r := range l {
-		if err := bc.appendCap(ctx, r); err != nil {
-			return nil, err
+		if err := bc.RetrieveCap(ctx, r); err != nil {
+			log.Warningf("failed to get capabilities, error: %v", err)
```
Comment (zyyw) [inline src/controller/scanner/base_controller.go:84]: why this is `Warning` instead of `Error`?

File: src/controller/scanner/base_controller.go [hunk 2]
```diff
@@ -129,13 +130,14 @@ func (bc *basicController) GetRegistration(ctx context.Context, registrationUUID
 	if r == nil {
 		return nil, nil
 	}
-	if err := bc.appendCap(ctx, r); err != nil {
-		return nil, err
+	if err := bc.RetrieveCap(ctx, r); err != nil {
+		log.Warningf("failed to get capabilities, error: %v", err)
```
Comment (zyyw) [inline src/controller/scanner/base_controller.go:134]: why this is `Warning` instead of `Error`?

File: src/server/v2.0/handler/project.go [hunk 3]
```diff
@@ -595,11 +595,9 @@ func (a *projectAPI) GetScannerOfProject(ctx context.Context, params operation.G
 		return a.SendError(ctx, err)
 	}
 	if scanner != nil {
-		metadata, err := a.scannerCtl.GetMetadata(ctx, scanner.UUID)
-		if err != nil {
-			return a.SendError(ctx, err)
+		if err := a.scannerCtl.RetrieveCap(ctx, scanner); err != nil {
+			log.Warningf("failed to retrieve scanner capabities, error %v", err)
```
Comment (zyyw) [inline src/server/v2.0/handler/project.go:599]: why this is `Warning` instead of `Error`?

File: src/controller/scanner/base_controller.go (hunk 1 above)
Comment (stonezdj) [reply src/controller/scanner/base_controller.go]: User could continue other works when the log level is warn

**Pull request merged**